### PR TITLE
syncthing: add -no-browser example

### PR DESCRIPTION
**Pull request by summonholmes**
I know this PR puts this page at 8 examples, but please consider adding it.  For those of us using Linux, this example can be set to auto-start upon login, removing the overhead of opening a web browser while other processes are starting up.  

Unfortunately, starting the daemon in the background is often not done by default in most distributions.

- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by navarroaxel** [comment]
@summonholmes You should sign the `license/cla` check and amend your commit using:

```bash
git commit --amend --author 'Your name <user@example.com>'
git push --force
```

the name and email must match your GitHub account then you pass the `license/cla` check.

**Comment by summonholmes** [comment]
Ah didn't realize name and email weren't configured on this laptop.  Thanks!

**Review by navarroaxel — APPROVED** [review]
@summonholmes The change is OK because the page has 8 examples and the limit is up to 8.

File: pages/common/syncthing.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `syncthing`
 
+- Start syncthing without opening a browser:
```
Comment (sbrl) [inline pages/common/syncthing.md:10]: Perhaps we should be slightly more specific here?
```suggestion
- Start syncthing without opening a web browser:
```

Comment (summonholmes) [reply pages/common/syncthing.md]: Absolutely agree, done!

**Review by einverne — APPROVED** [review]
LGTM.

**Pull request merged**